### ISSUE TITLE
Error - python unable to find webdriver

### ISSUE THREAD

**Issue by travel-bookph** [luanon404/Selenium-On-Termux-Android#5]
Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/selenium_manager.py", line 123, in run
    completed_proc = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^         File "/data/data/com.termux/files/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:         ^^^^^^^^^^^^^^^^^^^^^^^^^^^              File "/data/data/com.termux/files/usr/lib/python3.11/subprocess.py", line 1026, in __init__       self._execute_child(args, executable, preexec_fn, close_fds,
  File "/data/data/com.termux/files/usr/lib/python3.11/subprocess.py", line 1948, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)                                OSError: [Errno 8] Exec format error: '/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/linux/selenium-manager'
                                                The above exception was the direct cause of the following exception:
                                                Traceback (most recent call last):                File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/driver_finder.py", line 38, in get_path              path = SeleniumManager().driver_location(options) if path is None else path                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/selenium_manager.py", line 90, in driver_location
    output = self.run(args)                                  ^^^^^^^^^^^^^^                       File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/selenium_manager.py", line 129, in run               raise WebDriverException(f"Unsuccessful command executed: {command}") from err              selenium.common.exceptions.WebDriverException: Message: Unsuccessful command executed: /data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --output json
                                                                                                The above exception was the direct cause of the following exception:                                                                            Traceback (most recent call last):
  File "/data/data/com.termux/files/home/1.py", line 6, in <module>                                 driver = webdriver.Chrome(options=options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^    File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__                  super().__init__(                             File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py", line 51, in __init__                self.service.path = DriverFinder.get_path(self.service, options)                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/data/com.termux/files/usr/lib/python3.11/site-packages/selenium/webdriver/common/driver_finder.py", line 41, in get_path
    raise NoSuchDriverException(msg) from err   selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome using Selenium Manager.; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location

I thoroughly followed the instruction.
I followed option 1
Tried to do both chrome and firefox, it has the same issue.
I am using android 13 non root.
I hope you can help me

**Comment by luanon404** [comment]
which code did you run?

**Comment by travel-bookph** [comment]
This one - Using Selenium with a Headless Desktop WebDriver.

**Comment by luanon404** [comment]
did u install all requiments?

**Comment by luanon404** [comment]
looks like something wrong with package not your phone

**Comment by luanon404** [comment]
let me try again, pls wait

**Comment by travel-bookph** [comment]
Thank you for looking on to this

**Comment by luanon404** [comment]
> Thank you for looking on to this

which selenium version are you using?

**Comment by luanon404** [comment]
if your selenium > 4.9.x then uninstall and reinstall ```pip install selenium==4.9.1``` then try again

**Comment by travel-bookph** [comment]
Finally. It is resolved! Thanks
